UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -1260,6 +1147,100 @@ protected virtual void LethalElectrocution(Electrocution electrocution, float sh
 		}
 
 		#endregion
+
+				/// <summary>
+		/// The list of the internal net ids of the body parts contained within this container
+		/// </summary>
+		[Tooltip("The internal net ids of the body parts contained within this")]
+		public List<uint> InternalNetIDs;
+
+
+		public RootBodyPartController rootBodyPartController;
+
+
+		/// <summary>
+		/// Sets up the sprite of a specified body part and adds its Net ID to InternalNetIDs
+		/// </summary>
+		/// <param name="implant">Body Part to display</param>
+		public void SetupSpritesNetId(BodyPart implant)
+		{
+			int i = 0;
+			bool isSurfaceSprite = implant.IsSurface;
+			var sprites = implant.GetBodyTypeSprites(playerSprites.ThisCharacter.BodyType);
+			foreach (var Sprite in sprites.Item2)
+			{
+				var newSprite = Spawn.ServerPrefab(implant.SpritePrefab.gameObject, Vector3.zero, playerSprites.BodySprites.transform).GameObject.GetComponent<BodyPartSprites>();
```

Comment:
note Might have to adjust some sprite rendering orders Since if it'll been thrown on the same object misses separate objects before it might change the rendering order